webview/src/plots/components/TooManyPlots.tsx

```diff
@@ -10,11 +8,7 @@ const PathHighlight: React.FC<PropsWithChildren> = ({ children }) => (
 )
 
 export const TooManyPlots: React.FC = () => {
-  const shouldShow = useSelector(
-    (state: PlotsState) => state.webview.shouldShowTooManyPlotsMessage
-  )
-
-  return shouldShow ? (
+  return (
     <MessageBand icon={Info} id="too-many-plots-message">
```

Comment:
Minor! Have we considered adding a border with extra spacing to the message? 

<img width="1042" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/5c910624-75b6-44fc-9cc5-6b80d18e6f8f">

Personally, I think it looks better but up to you!